packages/bpk-tokens/src/ios/base/shadows.json

```diff
@@ -0,0 +1,82 @@
+{
+  "imports": [
+    "./aliases.json"
+  ],
+  "props": {
+    "SHADOW_COLOR_SM": {
+      "value": "{!GRAY_900}",
+      "type": "color",
+      "category": "shadowsettings"
```

Comment:
This category field is used (amongst other things) to pull values into the tokens page (see web example with `"box-shadows"` category value http://backpack.prod.aws.skyscnr.com/tokens/shadows).

2 options:

1. Use `"box-shadows"` - the same as web
1. Use a different value (perhaps just `"shadows"` tho :P) and update the `ShadowsPage` (https://github.com/Skyscanner/backpack/blob/master/packages/bpk-docs/src/pages/ShadowsPage/ShadowsPage.js#L29) to include them

Reply:
I like option one the most because is the same as web, unless we want to behave differently from native vs web, I'd keep it consistent; if we want to start to have different behaviour for RN option 2 will be a good starting point

Reply:
Yeah I agree - lets go with option 1. 

Related to this, I think we should update the shadows page to include `"elevation"` to surface the android tokens?